ckcp/openshift_dev_setup.sh

```diff
@@ -277,7 +277,7 @@ patches:
       kcp_image_tag="$(yq -e '.images[0].newTag' "$ckcp_dev_dir"/kustomization.yaml)"
       KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workload sync local \
       --syncer-image ghcr.io/kcp-dev/kcp/syncer:$kcp_image_tag \
-      --resources conditions.tekton.dev,pipelines.tekton.dev,pipelineruns.tekton.dev,pipelineresources.tekton.dev,runs.tekton.dev,tasks.tekton.dev,taskruns.tekton.dev,repositories.pipelinesascode.tekton.dev > "$kube_dir/syncer.yaml"
+      --resources conditions.tekton.dev,pipelines.tekton.dev,pipelineruns.tekton.dev,pipelineresources.tekton.dev,tasks.tekton.dev,runs.tekton.dev,repositories.pipelinesascode.tekton.dev > "$kube_dir/syncer.yaml"
```

Comment:
`ckcp` is only a local Dev Env. I think the changes needed wrt adding Pruners should only be made at places relevant to the end customer/user. Please revert the changes made in this script.

Reply:
cc @adambkaplan

Reply:
@ramessesii2 Yes, although its a dev environment, we could have the changes implemented in ckcp as well to maintain consistency. 
(note that by default pruner is anyways present in tekton even in our ckcp model and this change will modify it to prune just TaskRuns)